examples/SelfEncryptorExample.rs

```diff
@@ -0,0 +1,55 @@
+extern crate self_encryption;
```

Comment:
This is importing the main lib into the example. Users can run examples with `cargo run --example selfencryptorexample`